xgi/classes/simplicialcomplex.py

```diff
@@ -186,12 +186,16 @@ def add_simplex(self, simplex, **attr):
             faces = self._subfaces(simplex)
             self.add_simplices_from(faces)
 
-    def _subfaces(self, simplex):
+    def _subfaces(self, simplex, all=True):
```

Comment:
We forgot to document the new argument. A docstring a bit more verbose would useful too.

Reply:
Done